examples/Orekit_example/orekit_integration.ipynb

```diff
@@ -0,0 +1,256 @@
+{
```

Comment:
<blockquote>If everything went well, you should see the same plot again, except this time it's happening inside PASEOS. :)</blockquote><p>If everything went well, you should see the same plot again, except this time it's happening inside PASEOS :).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>